### PR TITLE
[757890] support for print stylesheet

### PR DESCRIPTION
**Pull request by Mte90**
Fix https://bugzilla.mozilla.org/show_bug.cgi?id=757890

### CODE REVIEW

File: remo/base/templates/base_fd4.html [hunk 1]
```diff
@@ -79,11 +79,11 @@
 
             <!-- Tabzilla Tab -->
             <div id="tabzilla">
-              <a href="https://www.mozilla.org/">Mozilla</a>
+              <a class="hide-on-print" href="https://www.mozilla.org/">Mozilla</a>
```
Comment (comzeradd) [inline remo/base/templates/base_fd4.html]: I'd move this to the div element right above.

File: remo/base/templates/base_fd4.html [hunk 2]
```diff
@@ -139,7 +139,7 @@
                   <img src="{{ static('base/img/remo/remo_logo_medium.png') }}"
                        alt="Mozilla Reps logo">
                   {% block logo %}
-                  <h2><span class="metabold">mozilla</span> reps</h2>
+                  <h2 class="hide-on-print"><span class="metabold">mozilla</span> reps</h2>
```
Comment (comzeradd) [inline remo/base/templates/base_fd4.html]: I'd prefer we have the wordmark when printing.

File: remo/base/templates/base_fd4.html [hunk 3]
```diff
@@ -243,7 +243,7 @@
 
       <!-- Client-side messages -->
       <div class="row">
-        <div id="client-message-container" class="hidden large-12 columns">
+        <div id="client-message-container" class="hidden large-12 columns hide-on-print">
```
Comment (comzeradd) [inline remo/base/templates/base_fd4.html]: Same as before, it's better to move this to the div row element above.

File: remo/base/templates/includes/footer_fd4.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <footer id="colophon">
-  <div class="row">
+  <div class="row hide-on-print">
```
Comment (comzeradd) [inline remo/base/templates/includes/footer_fd4.html]: Same here. You can move this to the footer element.

**Comment by comzeradd** [comment]
Hey Daniele,

Thanks for your contributions. This looks much nicer for printing. I left some inline comments for small fixes on your code.

**Comment by Mte90** [comment]
improvements done!

**Comment by comzeradd** [comment]
Can you squash all commits to one?

**Comment by Mte90** [comment]
I've merged the commit in one commit :-)

**Comment by comzeradd** [comment]
Great :) On last thing you can also amend the commit message to remove the old ones.

Something like this should do the trick:
`git commit --amend -m "[Fix bug 757890] support for print stylesheet"`

and then push -f again

**Comment by Mte90** [comment]
tadah!

**Comment by comzeradd** [comment]
Thanks! :rocket:

**Pull request merged**